data/03-10-svm.php

```diff
@@ -45,13 +45,453 @@
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>2</td>
+							<td>8</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>3</td>
+							<td>6</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>8</td>
+							<td>3</td>
+							<td>-1</td>
+						</tr>
+						<tr>
+							<td>7</td>
+							<td>4</td>
+							<td>-1</td>
+						</tr>
+						<tr>
+							<td>10</td>
+							<td>2</td>
+							<td>-1</td>
+						</tr>
+						<tr>
+							<td>10</td>
+							<td>5</td>
+							<td>-1</td>
+						</tr>
+						<tr>
+							<td>8</td>
+							<td>7</td>
+							<td>-1</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<!-- Tabela [fim] -->
+		</section>
+
+			<section class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
+				<p>
+					Se representarmos essa base de dados graficamente, veremos que os dados são <strong>linearmente separáveis</strong>. 
+					Isso significa que uma linha reta (hiperplano) pode ser desenhada de modo que os objetos da classe +1 
+					fiquem de um lado e os da classe -1 do outro (dizemos que a reta gera um modelo com erro de treinamento igual a 0). 
+					Mais precisamente, infinitos hiperplanos separadores poderiam ser desenhados como mostra a Figura 13. 
+					Todos esses hiperplanos classificam perfeitamente os objetos de treinamento. Porém, não há garantias de que 
+					a mesma performance se repetirá para novos objetos. Desta forma, o <span class="small-caps">SVM</span> deverá escolher um deles, 
+					baseado em quão bem funcionará para novos objetos. 
+				</p>
+
+		<!-- Imagem média [inicio] -->
+		<figure class="img-container">
+		<div
+			class="col-xs-22 col-xs-offset-1 col-sm-14 col-sm-offset-3 col-md-11 col-md-offset-5 col-lg-8 col-lg-offset-7 gutter-sm-right"
+		>
+			<picture>
+			<source
+				srcset="dist/img/image17.png"
+				media="(min-width:1200px)"
+			/>
+			<source
+				srcset="dist/img/image17.png"
+				media="(min-width:600px)"
+			/>
+			<source
+				srcset="dist/img/image17.png"
+				media="(min-width:1px)"
+			/>
+			<img
+				class="image"
+				src="dist/img/image17.png"
+				alt="Um classificador Random Forest ."
+			/>
+			</picture>
+		</div>
+		<figcaption
+			class="col-xs-22 col-xs-offset-1 col-sm-4 col-sm-offset-0 col-md-3 col-lg-2 img-container__caption"
+		>
+			<div class="img-container__top-line"></div>
+			Figura 13. Infinitos hiperplanos separadores com erro de treinamento igual a zero.
+		</figcaption>
+		</figure>
+		<!-- Imagem média [fim] -->
+		</section>
+
+		<section class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
+			<p>
+				Mas então, qual hiperplano será escolhido? A resposta é: o <span class="small-caps">SVM</span> irá procurar 
+				pelo <strong>hiperplano de margem máxima</strong> (<i>Maximum Margin Hyperplane<i> – <span class="small-caps">MMH</span>). Basicamente é o hiperplano que consegue 
```

Comment:
Faltou "fechar" a tag de itálico.

```suggestion
				pelo <strong>hiperplano de margem máxima</strong> (<i>Maximum Margin Hyperplane</i> – <span class="small-caps">MMH</span>). Basicamente é o hiperplano que consegue 
```